src/components/info/feature-info.vue

```diff
@@ -0,0 +1,89 @@
+<script setup lang="ts">
+import { defineProps, onUnmounted } from 'vue'
+import DefaultTemplate from '@/components/info/templates/default-template.vue'
+import LignesBusTemplate from './templates/lignes-bus-template.vue'
+import {
+  FeatureInfoJSON,
+  FeatureJSON,
+} from '../../services/info/feature-info.model'
+import { featureInfoLayerService } from '../../services/info/feature-info-layer.service'
+import {
+  exportFeatureService,
+  ExportFormat,
+  FeatExport,
+} from '@/services/export-feature/export-feature.service'
+import { Feature } from 'ol'
+import { Geometry } from 'ol/geom'
+import GeoJSON from 'ol/format/GeoJSON'
+import useMap, { PROJECTION_LUX } from '@/composables/map/map.composable'
+import { useFeatureInfoStore } from '@/stores/feature-info.store'
+
+defineProps({
+  content: {
+    type: Array as () => FeatureInfoJSON[],
+    required: true,
+  },
+})
+const map = useMap().getOlMap()
+const { clearContent } = useFeatureInfoStore()
+
+onUnmounted(() => {
+  clearContent()
+  featureInfoLayerService.clearFeatures()
```

Comment:
This clearFeatures is done 3 times in the composable, maybe it would be better to keep this one in the composable also (you can add onunmouted in composables).

Also, why not doing a reset() here?

Reply:
The composable is actually initiated in the hierarchy above the vue in the `map.container`, since the feature info events also need to be detected when the feature info vue is closed. So I don't think I can move the `onUnmounted` in this case.

Regarding the `reset`, it is not doing the same thing, but resetting evrything for new feature info to be displayed. I've renamed it to `done` to make this clearer I hope.